chimp.md

```diff
@@ -0,0 +1,213 @@
+
+- https://www.linkedin.com/in/akshaynassa/
+- https://ae.linkedin.com/in/rahulchaudhary1
+- https://ae.linkedin.com/in/saurabh-sharma-8540a8282
+- https://www.linkedin.com/in/tejas-chitnis/
+- https://www.linkedin.com/in/nidhi-nassa-030901306/
+- https://www.linkedin.com/in/pratik-bhuyan-0bb42079/
+- https://www.linkedin.com/in/jeeveshkuril/
+- https://www.linkedin.com/in/khushi-arora-84964222b/
+
+
+## Development Status :open_book:
+
+- Whitepaper : WIP
+- System Architecture graphic: https://drive.google.com/file/d/1jCoRayv_8ePR_xET5Nef3N3ge5Thc8el/view?usp=sharing
+- Technical Stack: JavaScript, NodeJS, ReactJS, VueJS, Typescript, Hardhat, Solidity
+
+## Development Roadmap :nut_and_bolt:
+
+
+### Overview
+
+- **Total Estimated Duration:** 3 months 
+- **Full-Time Equivalent (FTE):**  N/A
+- **Total Costs:** 30,000 USD
+
+### **Milestone 1: Private Trading Engine**
+
+| **Number**       | **Deliverable**                                    | **Details**                                                                                                                                                                            |
+|------------------|----------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **0a.**          | License                                            | Apache 2.0                                                                                                                                                                              |
+| **0b.**          | Documentation                                      | We will provide both **inline documentation** of the code and a basic **tutorial** explaining how users can execute private trades using the Chimp DEX private trading engine.           |
+| **0c.**          | Testing and Testing Guide                          | Core functions, including trade execution and encryption, will be covered by comprehensive unit tests. A **testing guide** will describe how to run these tests.                        |
+| **0d.**          | Docker                                             | We will provide a **Dockerfile** to set up the private trading engine and test private trade functionality on a local development environment.                                           |
+| **0e.**          | Article/Workshop                                   | We will publish an article explaining the significance of private trading in DeFi and showcasing how our **end-to-end encryption (E2EE)** works to provide anonymous trading.           |
+| **1.**           | Substrate module: Private Trading Engine           | We will create a Substrate module that implements the core **private trading engine**, enabling anonymous trade execution using **end-to-end encryption**.                               |
```

Comment:
This substrate pallet would need to be strongly specified somewhere in the application so that we actually understand what you want to do and how you envision it to work.